src/style.css

```diff
@@ -0,0 +1,45 @@
+* {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  width: 96px;
+  height: 96px;
+  justify-content: space-between;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+
+}
+
+.stars .stars__star {
```

Comment:
Don't increase specificity using 2 classes in one selector
Use only .stars__star class

Reply:
fixed